media/plg_system_tour/js/tours.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    $(document).ready(function() {
```

Comment:
If it has the features they need then it's fine.  But right now we as an overall project are not in a state where we can pick dependencies based on what their dependency chains are.  Frankly if you, Charlie, and Ciaran go away there aren't many left (if any) contributing actively who can maintain vanilla JavaScript resources, so we have to be careful here about balancing philosophical (Web 2017 and all the fun things you keep pushing for) with practical.